mmdet3d/datasets/pipelines/test_time_aug.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, results):
                 different scales and flips.
         """
         aug_data = []
-        flip_aug = [False, True] if self.flip else [False]
+        flip_aug = [True] if self.flip else [False]
```

Comment:
Is this modification necessary?

Reply:
Yes, it will reduce unnecessary scenes when using double flip augmentation during test time.

Reply:
We need a comment here as a remark to indicate other developers.